### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by IlyaMoskva**
Badges added to control build process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
 # degitx-simulator
 Simulation system for DeGItX
 
+[![Build Status](https://github.com/cqfn/degitx-simulator/workflows/Build%20degitx-simulator/badge.svg)](https://github.com/cqfn/degitx-simulator)
+![Build white paper document](https://github.com/cqfn/degitx-simulator/workflows/Build%20white%20paper%20document/badge.svg)
```
Comment (kirill-h5i) [inline README.md:5]: @IlyaMoskva maybe we can add a link from this badge to WP document uploaded? https://central.artipie.com/degit/wp/simulator-latest.pdf

**Review by kirill-h5i — CHANGES_REQUESTED** [review]
@IlyaMoskva please check my comment

File: README.md (hunk 1 above)
Comment (IlyaMoskva) [reply README.md:5]: @g4s8, I did it in a same way as all cqfn projects. Moreover, next line we have link with description what is pretty clean.

**Review by g4s8 — APPROVED** [review]
@IlyaMoskva ok, thanks

**Pull request merged**